bindings/cpp/src/lib.rs

```diff
@@ -20,6 +21,11 @@ mod uint;
 
 mod ethereum;
 
+#[derive(Serialize, Deserialize, Debug)]
```

Comment:
does it need `Debug`? maybe it can have a custom debug that won't print out the struct content, so that the mnemonic/password aren't accidentally leaked in logs?